apps/retrospective/services/sprint_task.go

```diff
@@ -311,3 +323,20 @@ func (service SprintTaskService) tasksWithTimeDetailsForCurrentAndPrevSprint(ret
 func (service SprintTaskService) AssignPointsToSprintTask(sprintTaskID string, sprintID string) {
 	workers.Enqueuer.EnqueueUnique("assign_points_to_sprint_task", work.Q{"sprintID": fmt.Sprint(sprintID), "sprintTaskID": fmt.Sprint(sprintTaskID)})
 }
+
+func RemoveDuplicatesFromSlice(list []string) string {
+	cleaned := []string{}
+	m := make(map[string]bool)
+
+	for _, item := range list {
+		if item == "" {
+			continue
+		}
+    if _, ok := m[item]; !ok {
+      m[item] = true
+      cleaned = append(cleaned, item)
```

Comment:
Shouldn't we take care of spaces here?

Reply:
Trim leading and trailing whitespaces, although they won't come in our query result. But since this is a utility function, we should consider this case. Thanks!